youtube_dl/extractor/livestreamfails.py

```diff
@@ -0,0 +1,49 @@
+from .common import InfoExtractor
+import json
+import time
+import calendar
+
+
+class LivestreamfailsIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?livestreamfails\.com/clip/(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'https://livestreamfails.com/clip/139200',
+        'md5': '8a03aea1a46e94a05af6410337463102',
+        'info_dict': {
+            'id': '139200',
+            'ext': 'mp4',
+            'display_id': 'ConcernedLitigiousSalmonPeteZaroll-O8yo9W2L8OZEKhV2',
+            'title': 'Streamer jumps off a trampoline at full speed',
+            'creator': 'paradeev1ch',
+            'thumbnail': 'https://livestreamfails-image-prod.b-cdn.net/image/3877b1d38db083fa25c82685bbaf645637e575ea.png',
+            'timestamp': 1656271785,
+            'upload_date': '20220626',
+        }
+    }]
+
+    def _real_extract(self, url):
+        result = {}
+        result['id'] = self._match_id(url)
```

Comment:
The normal style is to stash results in variables and then construct the result dict as it's about to be returned. But if you're using this style, why not do:
```suggestion
        result = {
            'id': self._match_id(url),
        }
```